douane.c

```diff
@@ -1182,6 +1181,9 @@ static struct nf_hook_ops nfho_outgoing = {
   .hooknum  = NF_IP_LOCAL_OUT,
   .pf       = NFPROTO_IPV4,
   .priority = NF_IP_PRI_LAST,
+#if LINUX_VERSION_CODE <= KERNEL_VERSION(4,12,14)
+//  .dev	    = , // .dev is not used!
```

Comment:
This looks strange to me. If the `.dev` is never used (meaning across all Linux kernel versions) then remove the entire `if` otherwise uncomment it.